smac/runhistory/runhistory.py

```diff
@@ -314,7 +323,7 @@ def update_from_json(self, fn, cs):
         """
         new_runhistory = RunHistory(self.aggregate_func)
         new_runhistory.load_json(fn, cs)
-        self.update(runhistory=new_runhistory)
+        self.update(runhistory=new_runhistory, external_data=True)
```

Comment:
I checked the code, and the way it is treated right now is that the data used in a different pSMAC run is used in both the EPM building and intensify. My "fix" would make the data be used in the EPM building only, right? Didn't SMAC2 treat pSMAC data like this? I'm currently wondering about the implications. If intensify sees external data, it will take more time for a new configuration to become an incumbent, because the number of evaluated instance seed pairs for the current incumbent grows #workers times faster than usually, possibly making SMAC spend more time in intensify than it would usually do. Is this intended? Maybe we should run both versions against each other to see the effect of both?